### PR TITLE
fix: Variable identifier type error, fix #36

### LINKED ISSUE #36: Argument 1 passed to Casbin\Rbac\DefaultRoleManager\RoleManager::hasLink() must be of the type string, int given, called in /var/www/html/amitdeveloper28/vendor/casbin/casbin/src/Util/BuiltinOperations.php on line 440
I am getting this issue, when I used following steps to reproduce this issue.

1. I am using laravel Laravel Framework 6.20.38.
2. add  permission of logged user in laravel. Below are my code:   $email = Auth::user()->email;
        // adds permissions to a user
        \Enforcer::addPermissionForUser($email, '/photo/index', 'GET');
3. When, I hit this allow path, I am getting this error:
4. Argument 1 passed to Casbin\Rbac\DefaultRoleManager\RoleManager::hasLink() must be of the type string, int given, called in /var/www/html/amitdeveloper28/vendor/casbin/casbin/src/Util/BuiltinOperations.php on line 440

### PR DESCRIPTION
**Pull request by basakest**
fix #36

### CODE REVIEW

**Comment by hsluoyz** [comment]
@techoner

**Comment by hsluoyz** [comment]
@amitgoldy1 plz review

File: src/Middlewares/RequestMiddleware.php [hunk 1]
```diff
@@ -44,7 +44,7 @@ public function handle($request, Closure $next, ...$guards)
     protected function authorize(Request $request, array $guards)
     {
         $user = Auth::user();
-        $identifier = $user->getAuthIdentifier();
+        $identifier = strval($user->getAuthIdentifier());
         if (method_exists($user, 'getAuthzIdentifier')) {
             $identifier = $user->getAuthzIdentifier();
         }
```
Comment (techoner) [inline src/Middlewares/RequestMiddleware.php:50]: @basakest Please move it to this line .

**Review by techoner — CHANGES_REQUESTED** [review]

**Review by techoner — APPROVED** [review]

**Pull request merged**